netaddr.go

```diff
@@ -1190,6 +1187,11 @@ func (r IPRange) Overlaps(o IPRange) bool {
 // Index 0 contains the high bits, index 1 contains the low.
 type uint128 [2]uint64
 
+// and returns the bitwise AND or u and m (u&m).
```

Comment:
*for

Reply:
Good catch. I think I meant "of" actually.